packages/lwc-language-server/src/__tests__/lwc-server.test.ts

```diff
@@ -76,26 +81,44 @@ describe('handlers', () => {
     describe('#onCompletion', () => {
         it('should return a list of available completion items in a javascript file', async () => {
             const params: CompletionParams = {
-                textDocument: { uri },
+                textDocument: { uri: jsUri },
                 position: {
                     line: 0,
                     character: 0,
                 },
                 context: {
-                    triggerCharacter: '{',
+                    triggerCharacter: '.',
```

Comment:
This test covers the autocomplete that was already being performed and not tested. There's a list of [other](https://github.com/forcedotcom/lightning-language-server/blob/34a0257b8f0b6ddb9f8caf7a6e37741e329b81f9/packages/lwc-language-server/src/lwc-server.ts#L120) trigger characters that would also work here, some of which are also not terribly helpful, but are not actively annoying our users right now.

Reply:
I actually think we should make an additional WI spike to figure out what how and why those are included. They don't seem particularly helpful, and if anything are also excessively noisy.